external/fv3fit/tests/_regtest_outputs/test_sklearn_wrapper.test_predict_is_deterministic.out

```diff
@@ -1 +1 @@
-fe1edd06a4cdb3bbca2652fc980a4d65
```

Comment:
Had to update the shape of the data used for this test to be pre-stacked in training.